src/articles/ExtensionMessage.js

```diff
@@ -1,18 +1,22 @@
 import Modal from "../components/modal_shared/Modal";
-import Header from "../components/modal_shared/Header";
-import Heading from "../components/modal_shared/Heading";
-import Main from "../components/modal_shared/Main";
+import { Header } from "../components/modal_shared/Header.sc";
+import { Heading } from "../components/modal_shared/Heading.sc";
+import { Main } from "../components/modal_shared/Main.sc";
 import FullWidthImage from "../components/FullWidthImage";
-import Footer from "../pages/_pages_shared/Footer";
-import ButtonContainer from "../components/modal_shared/ButtonContainer";
-import GoToButton from "../components/modal_shared/GoToButton";
+import { Footer } from "../components/modal_shared/Footer.sc";
+import { ButtonContainer } from "../components/modal_shared/ButtonContainer.sc";
+import { Button } from "../pages/_pages_shared/Button.sc";
 import { getExtensionInstallationLinks } from "../utils/extension/extensionInstallationLinks";
 import { isSupportedBrowser } from "../utils/misc/browserDetection";
-import { isMobile } from "../utils/misc/browserDetection";
-import FileDownloadOutlinedIcon from "@mui/icons-material/FileDownloadOutlined";
+import {
+  isMobile,
+  runningInChromeDesktop,
+} from "../utils/misc/browserDetection";
+import ArrowForwardRoundedIcon from "@mui/icons-material/ArrowForwardRounded";
 import Feature from "../features/Feature";
 import LocalStorage from "../assorted/LocalStorage";
-import strings from "../i18n/definitions";
+import redirect from "../utils/routing/routing";
+import { getExtensionInstallationButtonContent } from "../utils/extension/extensionInstallationButtonContent";
```

Comment:
As we agreed here (https://github.com/zeeguu/web/issues/521) - this `ExtensionMessage.js` pop-up from `/articles` is to be deleted (probably in the next PR).

The `getExtensionInstallationLinks` and `getExtensionInstallationButtonContent` functions are **shared** and used by the **modal** - `SupportedNotification_NotInstalled.js` from `components/redirect_notification`, and the **onboarding page** - `InstallExtension.js` from `pages/onboarding` so I personally think they shouldn't be moved to the `/articles` folder. I propose that we could still keep them where they are: `utils/extension`, but they could be grouped in their own folder, called for example `extension_installation`

Reply:
Optionally, in the future (next PRs), I would also see the `getExtensionInstallationLinks` and `getExtensionInstallationButtonContent` merged into one function that returns an object containing the name of the browser store and links to it :) But first, I would need to check how intuitive it looks in practice.

Reply:
ok, it's more complicated than i thought :)
could we then move everything related to extension installation not in utils, but somewhere in /components? 
i just have this feeling of what's in `/utils` is something we didn't think enough about :)

Reply:
> ok, it's more complicated than i thought :)

Yes :) 

> could we then move everything related to extension installation not in utils, but somewhere in /components?
i just have this feeling of what's in /utils is something we didn't think enough about :)

We could move them to `/components`, but I'm unsure how we could allocate them there. These functions have very specific purpose, and are not components. If we group them in a folder, we need to name the folder very well so that it is clear that these functions represent shared behavior, instead of components.